catalan.ini

```
@@ -0,0 +1,670 @@
+Serris = "Serris"
+Serris_Text = "Un depredador amfibi que és capaç d'aconseguir altes velocitats, la qual cosa el converteix en l'animal més ràpid de SR388. La capa de pell en la seva part baixa serveix com a òrgan respiratori i és sensible a baixades sobtades de temperatura. Els seus múltiples ulls i cos segmentat li proporcionen una gran eficiència visual i de moviment.# #Quan Serris es desplaça a una nova localització, excava túnels per a canalitzar l'aigua des de la seva llar anterior per a prevenir la deshidratació. Aquest comportament probablement és el que va inundar el Centre de Distribució."
+Genesis = "Gènesi"
+Genesis_Text = "Un depredador formidable. Aquest animal cec s'ha adaptat per a caçar en completa foscor, confiant en els seus sentits especialitzats per a aguaitar a les seves preses. El seu cos ofereix poca protecció i, com a resultat, intentarà ocultar-se en les ombres. Utilitza la seva sang àcida com a projectils mitjançant autohemorràgia.# #Aquesta espècie sol caçar en grup, la qual cosa la converteix en una espècie altament reeixida amb poca necessitat d'evolucionar; el seu ADN suggereix que pertany al llinatge d'éssers vius més antic de SR388, concedint-li el seu nom."
+
+[Items]
+MissileTank = "Tanc de Míssils"
+MissileTank_Text = "Capacitat de míssils incrementada en 5"
+MissileTank_TextHard = "Capacitat de míssils incrementada en 2"
+SuperMissileTank = "Súper Míssils"
+SuperMissileTank_Text = "Prem | per a seleccionar, dany x5"
+PowerBombTank = "Bombes d'Energia"
+PowerBombTank_Text = "Prem | en mode Morfosfera per a activar"
+EnergyTank = "Tanc d'Energia"
+EnergyTank_Text = "Energia de l'Armadura incrementada en 100 unitats"
+Bombs = "Bombes"
+Bombs_Text = "Prem | en mode Morfosfera per a col·locar-les"
+PowerGrip = "Agafar"
+PowerGrip_Text = "Ara pots agafar-te a sortints"
+SpiderBall = "Aracnosfera"
+SpiderBallPress_Text1 = "Prem | en mode Morfosfera per a grimpar superfícies"
+SpiderBallPress_Text2 = "Prem | o | en mode Morfosfera per a grimpar superfícies"
+SpiderBallHold_Text1 = "Mantingues | en mode Morfosfera per a grimpar superfícies"
+SpiderBallHold_Text2 = "Mantingues | o | en mode Morfosfera per a grimpar superfícies"
+JumpBall = "Rotosalt"
+JumpBall_Text = "Prem | en mode Morfosfera"
+HiJump = "Botes de Salt Alt"
+HiJump_Text = "Augmenta l'altura màxima del Salt"
+VariaSuit = "Armadura Climàtica"
+VariaSuit_Text = "Redueix els danys en un 50%"
+SpaceJump = "Salt Espacial"
+SpaceJump_Text = "Salta contínuament en l'aire"
+SpeedBooster = "Acceleració"
+SpeedBooster_Text = "Corre de manera contínua per a iniciar l'Acceleració"
+ScrewAttack = "Atac espiral"
+ScrewAttack_Text = "Salta sobre els enemics per a infringir danys massius"
+GravitySuit = "Armadura Gravitatoria"

```

can someone of you test that this doesn't clip on 4:3? There wasnt that much space left for varia suit.

---

This is used in-game when getting the item no? What's the character limit?

We could use other wording variants for suits like "vestit" which is shorter.

```suggestion
GravitySuit = "Armadura Gravitatòria"
```

or

```suggestion
GravitySuit = "Vestit Gravitatori"
```

If we go with the later option I would change all the references of the word "armadura" and replace them with "vestit"

---

I would keep "armadura" since i belive it makes more sense